x-pack/plugins/apm/server/routes/alerts/rule_types/docs/params_property_apm_transaction_error_rate.yaml

```diff
@@ -0,0 +1,39 @@
+# @kbn-doc-linker partial
+
+params_property_apm_transaction_error_rate:
+  required:
+    - windowSize
+    - windowUnit
+    - threshold
+    - environment
+  properties:
+    serviceName:
+      type: string
+      description: The service name from APM
+    transactionType:
+      type: string
+      description: The transaction type from APM
+    transactionName:
+      type: string
+      description: The transaction name from APM
+    windowSize:
+      type: number
+      description: The window size
+      example: 6
+    windowUnit:
+      type: string
+      description: The window size unit
+      enum:
+        - "m"
+        - "h"
+        - "d"
+    environment:
+      type: string
+      description: The environment from APM
+    threshold:
+      type: number
+      description: The error rate threshold value
+    groupBy:
```

Comment:
nit: Can we also add an enum for the available fields

here are all the  available fields `["service.name", "service.environment", "transaction.type", "transaction.name"]`

These will be applied by default `["service.name", "service.environment", "transaction.type"]` are by default. The user can pass `transaction.name`

Reply:
Updated with https://github.com/elastic/kibana/pull/158676/commits/5cf8b5e1963c1543378ffd7e0b48704b052bc86b

Reply:
nice thanks